### DISCUSSION TITLE
How do I get FQDN to work when access WebUI especially with Wildcard SSL Cert?

### DISCUSSION THREAD

**Discussion by rtchinnachot** [Security-Onion-Solutions/securityonion#2961] [Unsupported Versions] [answered]
Long story short, basically, I have Security Onion installed on-prem and everything is working fine except that once I tried to install our company Wildcard cert "*.company.com" it's getting very confused. The purpose is to be able to access the Securityonion console with HTTPS.

I followed this #1766. 

I set up the Securityonion hostname as 'secon', although, we have our DNS point to 'secon.company.com' correctly every time we access the secon.company.com the WebUI shorten the DNS name to just 'https://secon/login/?flow=0b83a452-68f9-4ea7-9c8f-d82ba8854815#/' so the Wildcard cert is not working as expected. 

Can anyone shed some light?

**Comment by facyber** [comment]
Hey can you check [2897](https://github.com/Security-Onion-Solutions/securityonion/discussions/2897)  maybe? I had similar issue where nginx didn't worked with FQDN so I had to nake some changes manualy after installation. 

Then I followed instructions for SSL, but generally I generated it with some company procedure on different server and just migrated/coppied certificates to SO server.

**Reply by rtchinnachot** [reply]
I checked out on that and was actually tried putting in the `server_names_hash_bucket_size ` parameter but that wasn't helpful. This issue is that every time you type in `https://secon.companyname.com` it always redirects to just `https://secon/login/?flow=0b83a452-68f9-4ea7-9c8f-d82ba8854815#/` in this format, so the actual `domain` name suffix being left out as the result the Wildcard CA doesn't kick in.

**Comment by TOoSmOotH** [comment, marked as answer]
Change your url_base in the the global.sls to secon.companyname.com and restart.

**Reply by rtchinnachot** [reply]
Just want to confirm that this works perfectly as suggested by @TOoSmOotH. We are now able to access `https://secon.companyname.com` with no security warning.
